go/pserver/service_test.go

```diff
@@ -30,7 +30,7 @@ const (
 
 func TestServiceFull(t *testing.T) {
 	var cp pserver.Checkpoint
-	s, err := pserver.NewService(0, 1, "", nil, cp)
+	s, err := pserver.NewService(0, time.Hour, "", nil, cp)
```

Comment:
Set save interval to one hour will without invoke the checkpoint. Maybe in the case of `TestServiceFull `, we should run `checkpoint`, `recover from checkpoint` at least once to cover function testing.